build-release-tools/lib/PlatformClients.py

```diff
@@ -99,6 +99,8 @@ def release_box(self, atlas_version):
         resp = self.session.put(release_url)
         if resp.ok:
             print "Release version {0} successfully!".format(atlas_version)
+        elif resp.text.find("already") != -1 and resp.status_code == 422:
```

Comment:
@changev , you are quite right .
google 422, and it's what server understand what you said but can't process. so add text-matching is good idea.
can you *comment* inside/embeded the code ? otherwise, people can't find those rational once this PR is closed.

```
The 422 (Unprocessable Entity) status code means the server understands the content type of the request entity (hence a 415(Unsupported Media Type) status code is inappropriate), and the syntax of the request entity is correct (thus a 400 (Bad Request) status code is inappropriate) but was unable to process the...
```